src/node/SymmetricCurveNode.ts

```diff
@@ -0,0 +1,30 @@
+import { Mode } from '@/types';
+
+import PathNode from './PathNode';
+
+export default class SymmetricCurveNode extends PathNode {
+    x1: number;
+    y1: number;
+    x3: number;
+    y3: number;
+
+    constructor(
+        mode: Mode,
+        x1: number,
+        y1: number,
+        x3: number,
+        y3: number
```

Comment:
mógłbyś zrobić
`public y3: number` i wtedy nie musisz robić niżej `this.y3 = y3` i możliwe, że nawet nie musisz deklarować tego wyżej ale tego nie jestem pewien